tests/osgi-tests/src/test/java/org/wso2/carbon/identity/mgt/store/connector/jdbc/test/osgi/store/IdentityStoreTests.java

```diff
@@ -242,6 +242,24 @@ public void testListUsersByClaimOffsetAndLengthInADomain() throws IdentityStoreE
     }
 
     @Test(dependsOnGroups = {"addUsers"})
+    public void testListUsersByClaimsOffsetAndLength() throws IdentityStoreException {
+
+        Assert.assertNotNull(realmService, "Failed to get realm service instance");
+
+        List<Claim> claims = new ArrayList<>();
+        Claim claim1 = new Claim("http://wso2.org/claims", "http://wso2.org/claims/lastName", "Espinoza");
+        claims.add(claim1);
+        Claim claim2 = new Claim("http://wso2.org/claims", "http://wso2.org/claims/email", "dan@wso2.com");
+        claims.add(claim2);
+
+        List<User> users = realmService.getIdentityStore().listUsers(claims, 1, 1);
+
+        Assert.assertNotNull(users, "Failed to list the users.");
+        Assert.assertTrue(!users.isEmpty() && users.size() == 1 , "Number of users received in the response " +
```

Comment:
We check single condition on assertion, not complex conditions. e.g.
Assert.assertFalse(users.isEmpty(), "At least one user should exist");
Assert.assertEquals(users.size(), 1, "Can have only one user");

Reply:
Fixed with https://github.com/wso2-extensions/carbon-security-user-store-jdbc/pull/95